### PR TITLE
Tls secret change

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
Change to the correct secret for the environment

### CODE REVIEW

File: helm_deploy/values-production.yaml [hunk 1]
```diff
@@ -39,8 +39,10 @@ generic-service:
       cpu: "30m"
 
   ingress:
-    hosts: 
+    tlsSecretName: prison-visits-staff-cert
```
Comment (psoleckimoj) [inline helm_deploy/values-production.yaml:42]: `prison-visits-booking-staff-cert` ?

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: helm_deploy/values-production.yaml (hunk 1 above)
Comment (mattops) [inline helm_deploy/values-production.yaml:42]: this needs to be `prison-visits-staff-justice-cert`

File: helm_deploy/prison-visits-2/values.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ generic-service:
 
   ingress:
     enabled: true
-    tlsSecretName: prison-visits-booking-cert
+    tlsSecretName: prison-visits-booking-staff-cert
```
Comment (mattops) [inline helm_deploy/prison-visits-2/values.yaml:24]: change this to `prison-visits-staff-justice-cert`

Comment (hutcheonb-moj) [reply helm_deploy/prison-visits-2/values.yaml:24]: prod/dev have different certificate names - would it be simpler to rectify this in CP environments to all be the same cert across environments?

And double check the host listed in that cert

**Pull request merged**